Makefile

```diff
@@ -43,7 +43,8 @@ endif
 OCF_ROOT ?= deps/iotivity-constrained
 JS ?= samples/HelloWorld.js
 VARIANT ?= release
-BLE_NAME ?= "Zephyr OCF Node"
+DEVICE_NAME ?= "Zephyr OCF Node"
+BLE_ADDR ?= "none"
```

Comment:
"none" doesnt get into the C code, its just for the Makefile.ocf_zephyr to check. If it finds "none" it wont define the ZJS_CONFIG_BLE_ADDRESS